containerd-shim/process.go

```diff
@@ -217,6 +217,18 @@ func (p *process) pid() int {
 	return p.containerPid
 }
 
+func (p *process) killAll() error {
```

Comment:
for now this should go in `process_linux.go` or it'll break `Solaris` implementation